internal/commands/pull/inputs.go

```diff
@@ -24,12 +24,18 @@ type inputs struct {
 	LocalPath           string
 	AppVersion          realm.AppConfigVersion
 	IncludeDependencies bool
+	IncludeNodeModules  bool
+	IncludePackageJSON  bool
 	IncludeHosting      bool
 	DryRun              bool
 	TemplateIDs         []string
 }
 
 func (i *inputs) Resolve(profile *user.Profile, ui terminal.UI) error {
+	if i.IncludeNodeModules && i.IncludePackageJSON {
+		return fmt.Errorf(`must not use both "--include-package-json" and "--include-node-modules" at the same time`)
```

Comment:
[nit] don't need `fmt.Errorf` if nothing's dynamic in the error message ... that being said, maybe we do want consts for `"include-package-json"` and `"include-node-modules"` in this file so we can be sure we are consistent with the names here
```suggestion
		return errors.New(`cannot use "--include-package-json" and "--include-node-modules" at the same time`)
```